k8s/aat/common/da/da-cos-api.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: da-cos-api
+  namespace: da
+spec:
+  releaseName: da-cos-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/da-cos-api
+  values:
+    java:
+      replicas: 2
+      memoryRequests: "768Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "3072Mi"
+      cpuLimits: "2500m"
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/da/cos-api:pr-*
```

Comment:
invalid image, use latest if you don't know what to put but it should have an image policy next to it

Reply:
updated to latest with image policy